addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/ui/ProgressPaneListener.java

```diff
@@ -48,6 +48,15 @@ protected ProgressPane getProgressPane() {
         return progressPane;
     }
 
+    /**
+     * Sets the total number of tasks to be processed.
+     *
+     * @param totalTasks the total number of tasks to be processed.
+     */
```

Comment:
```suggestion
     * @since 1.28.0
     */
```
The minimum version of `commonlib` in `exim.gradle.kts` needs to be updated and it can also be mentioned in the changelog under changed:
```
- Depend on newer version of Common Library add-on.
```